extensions/blocks/map/map-theme-picker/style.scss

```diff
@@ -16,6 +16,9 @@
 		background-size: contain;
 		transform: scale( 1 );
 		transition: transform 0.2s ease;
+		&:first-child, &:last-child {
```

Comment:
You can remove line 8 and unify to one group with `&, &:first-child,`

Reply:
👍🏻 b2b1e62b5